### ISSUE TITLE
Dialog height is too small

### ISSUE THREAD

**Issue by risalfajar** [vanpra/compose-material-dialogs#94]
I only added two items in List Dialog using `listItems()`, but only one item visible at once, when there's plenty of space left to show them at once.

![image](https://user-images.githubusercontent.com/28381124/126802946-3de22a06-17f5-47c0-a852-5e74f9929119.png)

```
    val chooserDialog = remember { MaterialDialog() }

    chooserDialog.build {
        title("Pilih jenis")
        listItems(list = listOf("Tool", "Material")) { index, item ->
            Timber.d("Clicked $item")
        }
    }
```

**Comment by risalfajar** [comment]
Turns out that removing `listRatio` from `listItems()` function work. May I ask what's the reason of using `listRatio`?

```
@Composable
fun MaterialDialog.listItems(
    list: List<String>,
    closeOnClick: Boolean = true,
    onClick: (index: Int, item: String) -> Unit = { _, _ -> }
) {
    BoxWithConstraints {
        val modifier = Modifier
            .heightIn(max = maxHeight * listRatio) //REMOVE listRatio
            .then(bottomPadding)

        //... rest of the code
    }
}
```

**Comment by vanpra** [comment]
Thanks for the bug report. The `listRatio` was used for limiting the height of the dialog as it would be off the screen in there are a lot of items in the list. I've already fixed this in the 0.5.0 branch by setting the limits on the size according to the material spec which I'll release as soon as I can.https://guides.github.com/features/mastering-markdown/

**Comment by risalfajar** [comment]
Then I'll close this for now, will be waiting for the next release, this library is great!
